specifications/api/swagger.yaml

```diff
@@ -752,6 +762,32 @@ components:
         outputs: *transactionOutputs
 
     ApiStakePool: &ApiStakePool
+      type: object
+      required:
+        - id
+        - relative_stake
+        - non_myopic_member_rewards
+        - produced_blocks
+        - apparent_performance
+        - cost
+        - margin
```

Comment:
On jormungandr we have a `metrics` object containing `produced_blocks` and `controlled_stake`. I don't see what is so special about those two fields compared to the rest, so I kept everything flat here.

Reply:
The rationale at the time was that what's in "metrics" was measured, whereas other parameters were simply read from the chain.

Reply:
Then why was not Apparent performance included?

Reply:
…and saturation, and desirability?

Reply:
Because many people modify many parts of the code and many of these fields were added in a rush resulting in an incoherent design.

Reply:
ok, then indeed, let's move all relevant fields to `metrics` (for shelley)